code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2408,32 +2408,27 @@
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-/datum/reagent/nanites
-	name = "Nanites"
-	id = NANITES
-	description = "Microscopic construction robots."
-	reagent_state = LIQUID
-	color = "#535E66" //rgb: 83, 94, 102
 
+
+/datum/reagent/nanites
```

Comment:
Convert these all back to tabs, you changed them to spaces by copy pasting from discord